docs/man/docker-build.1.md

```diff
@@ -30,6 +31,9 @@ When a Git repository is set as the **URL**, the repository is used
 as context.
 
 # OPTIONS
+**-f**, **--file**=*Dockerfile*
+   Location of the Dockerfile to use. The file must be within the build context. The default is *Dockerfile*.
```

Comment:
@duglin I don't think the docs here are clear on which it is - is `-f` relative to the `pwd`, or relative to the location of the context dir.

and as i note in the cli.md docs, if its relative to the `pwd` (which is how i read your docs atm), then the `-f` value will be different to the value sent to the daemon.